api/Api/Utilities/ControllableState.cs

```diff
@@ -0,0 +1,43 @@
+﻿using SynthesisAPI.Utilities;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Mirabuf.Signal;
+using Mirabuf;
+
+namespace SynthesisAPI.Utilities
+{
+    public class ControllableState
+    {
+        private Signals? _currentSignalLayout;
```

Comment:
If you could add some documentation to the controllable state that would be great. Just like a sentence or two of what it's supposed to do at the top